cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -175,29 +175,18 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     )
   }
 
-  @Test def updateGraph() {
+  @Test def createNode() {
     profileQuery(
-      title = "Update Graph",
+      title = "Create Node",
       text =
-        """Applies updates to the graph.""".stripMargin,
-      queryText = """CYPHER planner=rule CREATE (:Person {name: 'Alistair'})""",
+        """Creates a node in the graph.""".stripMargin,
+      queryText = """CREATE (:Person {name: 'Alistair'})""",
```

Comment:
I assume this change could (or should) have been made in 3.1 docs, right?

Reply:
It has now 👍